opentelemetry-exporter-cloud-trace/tests/test_cloud_trace_exporter.py

```diff
@@ -182,28 +185,102 @@ def test_extract_status(self):
             Status(details=None, code=2, message="error_desc"),
         )
 
-    def test_extract_attributes(self):
+    def test_extract_empty_attributes(self):
         self.assertEqual(
             _extract_attributes({}, 4), ProtoSpan.Attributes(attribute_map={})
         )
+
+    def test_extract_variety_of_attributes(self):
         self.assertEqual(
             _extract_attributes(self.attributes_variety_pack, 4),
```

Comment:
(this comment applies to all tests): Generally speaking you shouldn't assume that the reader of the test (or any code) knows what all the parameters are to the function you're calling. In this case `self.attributes_variety_pack` is a descriptive name, so it's clear to the reader that the first argument to `_extract_attributes` is some sort of collection of attributes. But the reader would have no idea what the `4` means nor whether it's important to the test unless they go and look up the definition of `_extract_attributes`.

There are two main ways to improve this: put the 4 in a variable/constant with a descriptive name, or (since Python allows you to name your parameters) simply replace the `4` with `num_attrs_limit=4`.

Reply:
Done! I've also made some other stuff into named arguments like the truncate_str args